.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "openwhisk"]
 	path = openwhisk
-	url = git@github.com:apache/openwhisk-deploy-kube.git
+	url = https://github.com/apache/openwhisk-deploy-kube.git
```

Comment:
This was necessary to get the GitHub pages deployment working for me (but had nothing to do with actually fixing the openethereum chart).

Reply:
Hmm yes I might remove the openwhisk chart as it causes bugs. People can refer to the official one for that use case.